src/olympia/bandwagon/views.py

```diff
@@ -381,7 +381,14 @@ def ajax_new(request):
 
     if request.method == 'POST' and form.is_valid():
         collection = form.save()
-        addon_id = request.POST['addon_id']
+
+        # addon_id is supplied as a GET argument for our ajax
+        # popup window.
```

Comment:
Couldn't we fix that instead ? It's weird to use `request.GET` in a block starting with `request.method == 'POST'` ...